addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13808,6 +13808,11 @@ msgctxt "#20419"
 msgid "Replace file names with library titles"
 msgstr ""
 
+#: system/settings/settings.xml
```

Comment:
Please add a verbal description for the purpose of the string, starting with `#. `.

Reply:
The string is just the menu item. Do I need to comment that??

Reply:
Yes. Translators need context so they can find best translations. They usually don't have coding skills to look it up in the code.